style/blockly.css

```
@@ -537,54 +508,54 @@ body[data-theme="low-vision"] {
 
 body .blocklyHtmlInput {
   font-family:
-    "Atkinson Hyperlegible Next", "Asap", Helvetica, Arial, Lucida, sans-serif !important;
+    'Atkinson Hyperlegible Next', 'Asap', Helvetica, Arial, Lucida, sans-serif !important;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Asap" (font-family-name-quotes)

(font-family-name-quotes)